tests/lib/linter/linter.js

```diff
@@ -13060,6 +13060,60 @@ var a = "test2";
                     assert.strictEqual(suppressedMessages[1].line, 3);
                 });
 
+                it("should report a violation with literals", () => {
```

Comment:
Can we use the term "quoted rule names" instead of "literals"?

"Literals" is confusing as it may refer to rule names that appear literally (without quotes), as used in https://github.com/eslint/eslint/issues/17598.